dlt/common/json/_orjson.py

```diff
@@ -21,7 +29,7 @@ def _dumps(
         options |= orjson.OPT_INDENT_2
     if sort_keys:
         options |= orjson.OPT_SORT_KEYS
-    return orjson.dumps(obj, default=default, option=options)
+    return orjson.dumps(obj, default=partial(default, _custom_encoder), option=options)
```

Comment:
as mentioned we'll skip partial here so this function goes back to the original. just a question: why `typed_dumpb` got skipped? you want to apply encoding to pua encoder as well (which is used between extract and normalize stages)

Reply:
> question: why `typed_dumpb` got skipped? you want to apply encoding to pua encoder as well (which is used between extract and normalize stages)

`typed_dumpb` delegates to `_dumps` which already curries here. If I were to curry there, then it would attempt to curry twice.